pkg/kubefed2/enable/enable.go

```diff
@@ -227,10 +228,14 @@ func GetResources(config *rest.Config, enableTypeDirective *EnableTypeDirective)
 // already been enabled for federation.  This would likely involve
 // updating the version of the target type and the validation of the schema.
 func CreateResources(cmdOut io.Writer, config *rest.Config, resources *typeResources, namespace string) error {
-	write := func(data string) {
+	write := func(data string) error {
```

Comment:
This is intended to be a braindead helper method.  It should not return an error and necessitate that every client checks for an error in turn.  Panicking should be sufficient.